src/ol/renderer/dom/domtilelayerrenderer.js

```diff
@@ -65,7 +64,9 @@ ol.inherits(ol.renderer.dom.TileLayer, ol.renderer.dom.Layer);
  * @inheritDoc
  */
 ol.renderer.dom.TileLayer.prototype.clearFrame = function() {
-  goog.dom.removeChildren(this.target);
+  while (this.target.lastChild) {
+    this.target.removeChild(this.target.lastChild);
+  }
```

Comment:
I'm wondering if it wouldn't make sense to create an `ol.dom.removeChildren` function, because this sequence here occurs multiple times.